Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM drupal:8.9.2-apache
+FROM drupal:8.9-apache AS test
```

Comment:
Should we continue to track the patch version of the image, as we saw a change which broke for us between `8.9.2` and `8.9.3`?

Reply:
This PR fixes the issue we had with that particular breaking change, not impossible something like that could happen again, but breaking our build would certainly bring it to our attention

Reply:
Sorry, hit "resolve" by mistake...

I think it's still sensible to pin to a patch version – that's the effect that `package.lock` has. The breakage we saw didn't show up in the Docker build (for me at least), so there's a risk that we could ship a build that broke production without us knowing about it 🙀.

Great work fixing that breakage so we can get up to 8.9.3 🙌. What was the problem?

Reply:
It was where Drupal was installed in the image, the change wasn't documented, but set up a new `WORKDIR` to install our stuff in to the correct place

Reply:
pinned to 8.9.6